src/node_platform.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_NODE_PLATFORM_H_
+#define SRC_NODE_PLATFORM_H_
+
+#include "libplatform/libplatform.h"
+#include "node_mutex.h"
+#include "uv.h"
+
+#include <queue>
+
+namespace node {
+
+using v8::Isolate;
+using v8::Platform;
+using v8::Task;
+
+class NodePlatform : public Platform {
+ public:
+  NodePlatform(int thread_pool_size, uv_loop_t* loop,
+               v8::TracingController* tracing_controller);
```

Comment:
This was discussed briefly in https://github.com/nodejs/node/issues/12980. This is not an attempt to define a general VM interface. The only goal is to provide a new implementation of `v8::Platform` as the default platform which is currently used by Node will may allow the process to exit during wasm module compilation.